packages/agent-utils-fs/src/FileSystemWorkspace.ts

```diff
@@ -66,6 +66,7 @@ export class FileSystemWorkspace implements Workspace {
   readdirSync(subpath: string): DirectoryEntry[] {
     const absPath = this.toWorkspacePath(subpath);
     return fs.readdirSync(absPath, { withFileTypes: true })
+      .filter((d) => !d.name.startsWith("."))
```

Comment:
Shouldn't it be ".evo"? Could the workspace have other `.` files that are relevant to the agent?

Reply:
I figured it's best-practice (and more generic) to ignore `.` files/directories since they are "hidden" by nature. This works for existing dot-files like `.git`.

Reply:
I understand, I just wonder if someone is building some project with evo and they create a .env file (or some other) it could cause issues